### ISSUE TITLE
301 Redirect not working when view is a param in old URL

### ISSUE THREAD

**Issue by travisbotello** [Sterc/SEOTab#179]
I am running MODx 2.7.1 with SEOTab 2.3.0 and Nginx and it seems like old URLs are not correctly redirected when one of the params is &view.

index.php?option=com_k2&id=104722 => working
index.php?option=com_k2&view=104722=> not working

Is this a SEO Pro related issue? Any ideas?

Thanks in advance!

**Comment by ic3m3n** [comment]
@travisbotello Hi, maybe this solves your problem? https://github.com/Sterc/SEOTab/issues/184

**Comment by travisbotello** [comment]
@ic3m3n No, that doesn't help. Actually when I apply your code both URLs are not redirecting.
